### PR TITLE
Updates steering members based on 2020 elections

### PR DESCRIPTION
**Pull request by bsnchan**
# Changes

Updates steering members based on 2020 elections in peribolos and steering-committee.md.

### CODE REVIEW

File: STEERING-COMMITTEE.md [hunk 1]
```diff
@@ -90,13 +90,11 @@ first name):
...
 | <img width="30px" src="https://github.com/thisisnotapril.png"> | April Kyle Nassi | Google       | [@thisisnotapril](https://github.com/thisisnotapril) |
 | <img width="30px" src="https://github.com/bsnchan.png">        | Brenda Chan      | VMware       | [@bsnchan](https://github.com/bsnchan)               |
 | <img width="30px" src="https://github.com/mbehrendt.png">      | Michael Behrendt | IBM          | [@mbehrendt](https://github.com/mbehrendt)           |
 | <img width="30px" src="https://github.com/pmorie.png">         | Paul Morie       | Red Hat      | [@pmorie](https://github.com/pmorie)                 |
-| <img width="30px" src="https://github.com/ronavn.png">         | Ron Avnur        | Google       | [@ronavn](https://github.com/ronavn)                 |
-|                                                                | Open \*          | Google       |                                                      |
+| <img width="30px" src="https://github.com/vaikas.png">         | Ville Aikas      | VMware       | [@vaikas](https://github.com/vaikas)                 |
 
 \* Open seats have been allocated to the organization per the prior committee
```
Comment (evankanderson) [inline STEERING-COMMITTEE.md:99]: Do we still need this note on open seats?

**Review by evankanderson — COMMENTED** [review]
/lgtm
/hold

One comment

**Comment by evankanderson** [comment]
Do you want to update OWNERS/OWNERS_ALIASES, too?

File: STEERING-COMMITTEE.md (hunk 1 above)
Comment (bsnchan) [reply STEERING-COMMITTEE.md]: good catch, this should be fixed now :)

**Comment by bsnchan** [comment]
@evankanderson good catch! I've updated the OWNER_ALIASES file, removed the comment around open seats and caught a a few more places that I initially missed in `peribolos/knative.yaml`

**Review by evankanderson — COMMENTED** [review]
/lgtm

**Review by evankanderson — COMMENTED** [review]
/hold cancel

**Pull request merged**